io.github.spacingbat3.webcord.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.spacingbat3.webcord
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+modules:
+  - name: webcord
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/vega-d/WebCord_flatpak_build/releases/download/v0.0-rolling/WebCord_tarball.tar.gz
+        sha256: ebc9ec1cb38819722b78705f83e5d07e32b8c75b4cf32f9bb72b94b447a463b8
+        dest: ./main
+      #- type: archive
+      #  path: ../WebCord_tarball.tar.gz
+      #  dest: ./main
+      - type: script
+        dest-filename: run.sh
+        commands:
+        - zypak-wrapper /app/bin/webcord "$@"
+      - type: file
+        path: ./vitamins/io.github.spacingbat3.webcord.desktop
+      - type: file
+        path: ./vitamins/io.github.spacingbat3.webcord.metainfo.xml
+      - type: file
+        path: ./vitamins/icon48.png
```

Comment:
so upstream has no icon and no desktop file?

I can see the lack of appdata, but the two other are actual requirement to pretend to support Linux

Reply:
Upstream has icons available [here](https://github.com/SpacingBat3/WebCord/tree/master/sources/assets/icons) but not all sizes. `.desktop` is not made available in the source tree anywhere at all.

Reply:
128x128 is the only requirement (or bigger or SVG)